runtime/Makefile

```diff
@@ -57,7 +57,7 @@ $(CHPL_ENV_HEADER):
 	@echo "/* THIS FILE IS AUTOMATICALLY GENERATED */" >> $(CHPL_ENV_HEADER)
 	@echo "#ifndef _CHPL_ENV_GEN_H_" >> $(CHPL_ENV_HEADER)
 	@echo "#define _CHPL_ENV_GEN_H_" >> $(CHPL_ENV_HEADER)
-	@$(CHPL_MAKE_HOME)/util/printchplenv --simple | \
+	@$(CHPL_MAKE_HOME)/util/printchplenv --all --no-tidy --anonymize --simple | \
```

Comment:
I know that this used to work with the equivalent of --all, but actually it really really should not print out compiler variables. Can we change it to --runtime --no-tidy --anonymize --simple? That will solve issue #6547.